src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Json.cs

```diff
@@ -0,0 +1,239 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.PowerFx.Core.App.ErrorContainers;
+using Microsoft.PowerFx.Core.Binding;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Syntax;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    // JSON(data:any, [format:s, [included_columns:*[Value:s]]])    
```

Comment:
>, [included_columns:*[Value:s] [](http://example.com/codeflow?start=31&length=30)

There is no 3rd argument to the JSON function

Reply:
Coming from PA Client :)
Fixed!